doc/DevelopersGuide/scheme_template.F90

```diff
@@ -7,24 +7,23 @@
 ! - scheme must be in its own module (module name = scheme name) and must
 !   have three entry points (subroutines) starting with the name of the module:
 !   module scheme_template -> subroutines scheme_template_{init,finalize,run}
+! 
+! - each .f or .F90 file with CCPP entry point scheme(s) must be accompanied by a 
+!   .meta file containing metadata for the scheme(s) 
 !
-! - empty schemes (e.g., scheme_template_init below) do not need an argument table
-!
-! - schemes in use require an argument table as below; order of arguments in the
-!   table must be the same as in the argument list of the subroutine
+! - empty schemes (e.g., scheme_template_init below) do not need metadata
```

Comment:
Does the prebuild still require empty schemes?

Reply:
Yes.